monorepo/monorepo-ws/apps/articleapp/src/Schema/article/index.test.ts

```diff
@@ -0,0 +1,54 @@
+import { gql } from "apollo-server";
```

Comment:
❓ je to ve slozce Schema 

mam dojem ze jsme to nikdy nemeli jen jako Schema a prosim vsechny slozky malym pismenem

Reply:
schema přejmenováno na resolvers jako v ukázkovém WS